Modules/modulesHelper.sh

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env bash
+set -e ;# exit on error
+set -u ;# exit when using undeclared variable
+#set -x ;# debugging
+
+DONOR=SkeletonDPL
+DONOR_LC=skeleton_dpl
+DONOR_TASK=SkeletonTaskDPL
+DONOR_TASK_INCLUDE_GUARD=QC_MODULE_EXAMPLE_EXAMPLETASKDPL_H
+DONOR_CHECK=SkeletonCheckDPL
+DONOR_CHECK_INCLUDE_GUARD=QUALITYCONTROL_LIBS_CHECKER_SkeletonCheckDPL_H_
```

Comment:
Same remark as above

Reply:
Is the convention `QC_MODULE_${DONOR}_${DONOR_CHECK}_H` fine with you? It resembles more the file structure.

Reply:
yes, it is.